spark/src/main/scala/org/apache/comet/serde/QueryPlanSerde.scala

```diff
@@ -2518,6 +2518,15 @@ object QueryPlanSerde extends Logging with CometExprShim {
           return None
         }
 
+        if (groupingExpressions.exists(expr =>
```

Comment:
Are there other check we need to include here for structs/arrays?

Reply:
hmm I suppose there could be structs containing maps 🤔